provider/win_driver_installer_test.py

```diff
@@ -269,270 +268,15 @@ def iozone_test(test, params, vm, images):
-    Run the virtiofs service on Windows guest
-    :param test: QEMU test object
-    :param params: Dictionary with the test parameters
-    :param session: the session of guest
-    """
-    create_viofs_service(test, params, session)
-    start_viofs_service(test, params, session)
-
-
-@error_context.context_aware
-def stop_viofs_service(test, params, session):
-    """
-    Stop the virtiofs service on Windows guest
-    :param test: QEMU test object
-    :param params: Dictionary with the test parameters
-    :param session: the session of guest
-    """
-    viofs_sc_stop_cmd = params.get("viofs_sc_stop_cmd", "sc stop VirtioFsSvc")
-    session.cmd(viofs_sc_stop_cmd)
-
-    test.log.info("Query status of the virtiofs service...")
-    output = query_viofs_service(test, params, session)
-    if "RUNNING" in output:
-        test.error("Virtiofs service is still running.")
-    else:
-        test.log.info("Virtiofs service is stopped.")
-
-
-def install_winfsp(test, params, session):
-    """
-    Install the winfsp on Windows guest
-    :param test: QEMU test object
-    :param params: Dictionary with the test parameters
-    :param session: the session of guest
-    """
-    cmd_timeout = params.get_numeric("cmd_timeout", 120)
-    install_path = params["install_winfsp_path"]
-    check_installed_cmd = params.get("check_winfsp_installed_cmd",
-                                     r'dir "%s" |findstr /I winfsp')
-    check_installed_cmd = check_installed_cmd % install_path
-    install_winfsp_cmd = r'msiexec /i WIN_UTILS:\winfsp.msi /qn'
-    install_cmd = params.get("install_winfsp_cmd",
-                             install_winfsp_cmd)
-    # install winfsp tool
-    test.log.info("Install winfsp for windows guest.")
-    installed = session.cmd_status(check_installed_cmd) == 0
-    if installed:
-        test.log.info("Winfsp tool is already installed.")
-    else:
-        install_cmd = utils_misc.set_winutils_letter(session, install_cmd)
-        session.cmd(install_cmd, cmd_timeout)
-        if not utils_misc.wait_for(lambda: not session.cmd_status(
-                check_installed_cmd), 60):
-            test.error("Winfsp tool is not installed.")
-
-
-def get_virtiofs_driver_letter(test, fs_target, session):
-    """
-    Get the virtiofs driver letter( Windows guest only )
-    :param test: QEMU test object
-    :param fs_target: virtio fs target
-    :param session: the session of guest
-    :return driver_letter: the driver letter of the virtiofs
-    """
-    test.log.info("Get driver letter of virtio fs target, the driver label is "
-                  "%s." % fs_target)
-    vol_con = "VolumeName='%s'" % fs_target
-    driver_letter = utils_misc.wait_for(
-        lambda: utils_misc.get_win_disk_vol(session, condition=vol_con),
-        timeout=120)
-    if driver_letter is None:
-        test.fail("Could not get virtio-fs mounted driver letter.")
-    return driver_letter
-
-
 def viofs_basic_io_test(test, params, vm):
```

Comment:
hi @nickzhq , I didn't find this function is now being used by any case, if it is, I think we can remove this function too